### DISCUSSION TITLE
offer/experience/2022summary

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#48] [Announcements]
# offer/experience/2022summary

千万不要因为走得太久，而忘记了我们为什么出发；2022也是波折比较多的一年，工作上换了组，工作内容发生翻天覆地的变化；生活中换了租处，换了新的装备；经历了防疫政策的突变，从🐑变为阳康，更多细节文中慢慢道来。。。

https://sugarat.top/offer/experience/2022summary.html

**Comment by cnjimbo** [comment]
sf
